docs/crds/gslbconfig.md

```diff
@@ -36,4 +36,9 @@ spec:
 ### Notes
 * Only one `GSLBConfig` object is allowed.
 * If using `helm install`, a `GSLBConfig` object is created by picking up values from the `values.yaml` file.
-* During `helm delete`, the `GSLBConfig` that holds the UUID of the current instance is deleted. Hence a cleanup of stale GSLB services, if any, is required at the controller before re-installing AMKO.
+* During `helm delete`, the `GSLBConfig` that holds the UUID of the current instance is deleted. To maintain the correct state of AMKO when you install AMKO again conserve the amkoUUID from annotations of GSLBconfig and add it in `configs.amkoUUID` field of [values.yaml](../../README.md#parameters). Otherwise a cleanup of stale GSLB services, if any, is required at the controller before re-installing AMKO.
```

Comment:
should we provide restoring of amkoUUID as step by step guide like `take backup of amkouuid,  use that amkouuid in values.yaml` ?

Reply:
I think this should be kept here . I will also add a detailed guide in troubleshooting section